relayer/relays/parachain/beefy-listener.go

```diff
@@ -34,6 +35,7 @@ type BeefyListener struct {
 	paraID              uint32
 	tasks               chan<- *Task
 	eventQueryClient    QueryClient
+	AccountID           *[32]byte
```

Comment:
Minor nit: Prefer we store the array here and not a pointer to the array:
```suggestion
	AccountID           [32]byte
```